docker-compose.yml

```diff
@@ -42,3 +42,16 @@ services:
   redis:
     image: redis:6-alpine
 
+  minio:
+    image: minio/minio:latest
+    command: server /export
+    ports:
+      - "${MINIO_PORT:-9000}:9000"
```

Comment:
I noticed recently that shared/worker are conflicting with each other, as they are exposing this port to the host. we don’t need this in order to run tests, so I would just not do this.